### PR TITLE
Adds Travis and publish settings

### PR DESCRIPTION
**Pull request by fedefernandez**
Please @juanpedromoreno could you review? Thanks

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 addSbtPlugin("com.fortysevendeg" % "sbt-catalysts-extras" % "0.1.2")
 addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "0.4.10")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```
Comment (juanpedromoreno) [inline project/plugins.sbt]: If you are using `sbt-catalysts-extras`, pgp plugin is included transitively.

File: build.sbt [hunk 2]
```diff
@@ -29,8 +24,54 @@ lazy val pluginSettings = Seq(
     scalafmtConfig in ThisBuild := Some(file(".scalafmt"))
   ) ++ reformatOnCompileSettings
 
-lazy val `sbt-dependency-updates` = (project in file("."))
+lazy val gitUrl = "https://github.com/47deg/sbt-dependencies"
+
+lazy val publishSettings = Seq(
```
Comment (juanpedromoreno) [inline build.sbt]: This settings is automatically included by `sbt-catalysts-extras`

File: build.sbt [hunk 3]
```diff
@@ -29,8 +24,54 @@ lazy val pluginSettings = Seq(
     scalafmtConfig in ThisBuild := Some(file(".scalafmt"))
   ) ++ reformatOnCompileSettings
 
-lazy val `sbt-dependency-updates` = (project in file("."))
+lazy val gitUrl = "https://github.com/47deg/sbt-dependencies"
```
Comment (juanpedromoreno) [inline build.sbt]: Not needed if you're using sbt-catalysts-extras

**Review by juanpedromoreno — CHANGES_REQUESTED** [review]

**Comment by fedefernandez** [comment]
Ready for a second review, thanks!

**Review by juanpedromoreno — APPROVED** [review]

**Comment by juanpedromoreno** [comment]
Great, thanks @fedefernandez !

**Pull request merged**